modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java

```diff
@@ -76,6 +76,10 @@ public DDMFormValues getDDMFormValues() {
 		return _ddmFormValues;
 	}
 
+	public long getFormId() {
```

Comment:
I believe it was because the json property is called [formId](https://github.com/natocesarrego/liferay-portal/pull/1014/files/cf20dd9b49c55c9e06a9e9bc2b15bbf6e937994a..ae817e21536fbe8d0d766d76ea085da3e43d2a72#diff-020463c3ef409b05a634588576d3f05023650b45c5abf61e4b4473f9e9a0ca5aR550).

Reply:
@brunofariasdeo can you change to `formInstanceId`? Thanks